bonus/inc/visuals.h

```diff
@@ -8,6 +8,14 @@
 #define MIN_TERMINAL_HEIGTH             9
 #define MIN_TERMINAL_WIDTH              MIN_BOARD_WIDTH + MIN_INFO_PANEL_WIDTH
 
+#define NO_GAME_MODE "Game mode is not selected yet"
```

Comment:
Can we remove the `is` from this message? Fits better to the others.

And it would also fix this issue (increasing the minimum width by 1 is also possible):

https://github.com/user-attachments/assets/df6c2124-aed5-4478-80b7-aed7367a72ac

```suggestion
#define NO_GAME_MODE "Game mode not selected yet"
```

Reply:
Sure we can, but I’ll also want to see when did I brake that.

Reply:
I think this is a logic error, in how we decide how many lines to print, that was always there. Just nothing was able to ever show it.

Reply:
I removed the is, but also fixed the multiline thing